composer.json

```diff
@@ -35,7 +35,7 @@
         }
     },
     "scripts": {
-        "test": "phpunit"
+        "test": "php vendor/phpunit/phpunit/phpunit"
```

Comment:
`vendor/bin/phpunit` is a PHP file but without `.php` extension

     #!/usr/bin/env php
      <?php

As [in the docs](http://php.net/manual/en/features.commandline.usage.php): "The special shebang first line for Unix does no harm on Windows"